### PR TITLE
Staging the 1.24.0 release

### PR DESCRIPTION
**Pull request by reyang**
This release will mark JVM metrics stable.

@trask do you want to include #612?

### CODE REVIEW

**Review by AlexanderWert — APPROVED** [review]

**Review by jsuereth — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,14 @@ release.
 
 ## Unreleased
 
+### Breaking
+
+### Features
+
+### Fixes
+
+## v1.24.0 (2023-12-15)
```
Comment (arminru) [inline CHANGELOG.md:16]: The following entries appear to be missing:

- https://github.com/open-telemetry/semantic-conventions/pull/499
- https://github.com/open-telemetry/semantic-conventions/pull/491
- https://github.com/open-telemetry/semantic-conventions/pull/493
- https://github.com/open-telemetry/semantic-conventions/pull/490
- https://github.com/open-telemetry/semantic-conventions/pull/573
- https://github.com/open-telemetry/semantic-conventions/pull/333

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,14 @@ release.
...
+
 - Replace AWS X-Ray Environment Span Link section with AWS X-Ray Active Tracing Considerations
   ([#354](https://github.com/open-telemetry/semantic-conventions/pull/354))
 - Move Exceptions to attribute registry
```
Comment (arminru) [inline CHANGELOG.md:20]: These should be under a heading.

**Review by arminru — APPROVED** [review]

**Comment by trask** [comment]
> @trask do you want to include #612?

It's not needed

File: CHANGELOG.md (hunk 1 above)
Comment (reyang) [reply CHANGELOG.md:16]: Thanks @arminru! I did a manual comparison and fixed all the entries I've discovered.

File: CHANGELOG.md (hunk 2 above)
Comment (reyang) [reply CHANGELOG.md]: I've moved the 1st one to Features and removed the 2nd one (since it seems to be just moving things around).

**Pull request merged**